app/src/androidTest/java/org/simple/clinic/rules/ServerAuthenticationRule.kt

```diff
@@ -24,6 +24,8 @@ import org.simple.clinic.security.PasswordHasher
 import org.simple.clinic.user.User
 import org.simple.clinic.user.UserSession
 import org.simple.clinic.user.UserStatus
+import org.simple.clinic.user.finduser.FindUserResult.*
```

Comment:
Not that important, can do this later. We are not using wildcard imports according to our code style, so can we break this down to individual imports, please?

Reply:
Oh yes, I usually set this to happen automatically when I format code, but it seems like it didn't run. Let me see what's going on.